workshops/chrome-extension/background.js

```diff
@@ -0,0 +1,74 @@
+let timer;
+
+chrome.browserAction.setBadgeBackgroundColor({ color: '#777' });
+
+// retrieve and format timer text or play audio if time is up
+function getTimeLeft(endTime) {
+  const distance = new Date(endTime) - new Date();
+  if (distance <= 0) {
+    timerEnded();
+    return '';
+  }
+  const hours = Math.floor(distance / (1000 * 3600));
```

Comment:
is there anyway you could use moment in the extension? lots of manual date parsing seems hard to me

Reply:
We were trying to avoid using third party libraries for the workshop, but we intend to have a google doc (or something) so they can copy paste some of the more annoying / tedious date logic